`ModifyProcessInstanceRejectionTest#shouldRejectActivationWhenAncestorScopeIsNotFlowScope` test fails on Windows

**Summary**

Run `ModifyProcessInstanceRejectionTest#shouldRejectActivationWhenAncestorScopeIsNotFlowScope` test failed on Windows 10.

**Logs**


<details><summary>Logs</summary>
<pre>
E PI   SEQ_FLOW_TAKEN    - #63->#05 K21 - SEQUENCE_FLOW "sequenc..b588006" in <process "process"[K03]>
C PI   ACTIVATE          - #64->#05 K22 - PARALLEL_GATEWAY "join" in <process "process"[K03]>
R PI   ACTIVATE          - #65->#05 K22 - PARALLEL_GATEWAY "join" in <process "process"[K03]> !INVALID_STATE (Expected to be able to activate parallel gateway 'join', but not all sequence flows have been taken.)
C MOD  MODIFY            - #66-> -1 K03 - <activate "A" no vars> <activate "B" no vars> <activate "A" no vars> 
R MOD  MODIFY            - #67->#66 K03 - <activate "A" no vars> <activate "B" no vars> <activate "A" no vars>  !INVALID_ARGUMENT (Expected to modify instance of process 'process' but it contains one or more activate instructions with an ancestor scope key that is not an ancestor of the element to activate:
- instance '2251799813685257' of element 'subProcess2' is not an ancestor of element 'A'
- instance '2251799813685263' of element 'B' is not an ancestor of element 'A')



java.lang.AssertionError: [Expect that subProcess2 cannot be selected as ancestor of task A] 
Expecting rejectionReason of:
  <{"valueType":"PROCESS_INSTANCE_MODIFICATION","key":2251799813685251,"position":67,"timestamp":1685115125924,"recordType":"COMMAND_REJECTION","intent":"MODIFY","partitionId":1,"rejectionType":"INVALID_ARGUMENT","rejectionReason":"Expected to modify instance of process 'process' but it contains one or more activate instructions with an ancestor scope key that is not an ancestor of the element to activate:\r\n- instance '2251799813685257' of element 'subProcess2' is not an ancestor of element 'A'\r\n- instance '2251799813685263' of element 'B' is not an ancestor of element 'A'","brokerVersion":"8.3.0","value":{"processInstanceKey":2251799813685251,"ancestorScopeKeys":[],"terminateInstructions":[],"activateInstructions":[{"ancestorScopeKeys":[],"elementId":"A","ancestorScopeKey":2251799813685257,"variableInstructions":[]},{"ancestorScopeKeys":[],"elementId":"B","ancestorScopeKey":2251799813685257,"variableInstructions":[]},{"ancestorScopeKeys":[],"elementId":"A","ancestorScopeKey":2251799813685263,"variableInstru...>
to be:
  <Expected to modify instance of process 'process' but it contains one or more activate instructions with an ancestor scope key that is not an ancestor of the element to activate:
- instance '2251799813685257' of element 'subProcess2' is not an ancestor of element 'A'
- instance '2251799813685263' of element 'B' is not an ancestor of element 'A'>
but was:
  <Expected to modify instance of process 'process' but it contains one or more activate instructions with an ancestor scope key that is not an ancestor of the element to activate:
- instance '2251799813685257' of element 'subProcess2' is not an ancestor of element 'A'
- instance '2251799813685263' of element 'B' is not an ancestor of element 'A'>

	at io.camunda.zeebe.engine.processing.processinstance.ModifyProcessInstanceRejectionTest.shouldRejectActivationWhenAncestorScopeIsNotFlowScope(ModifyProcessInstanceRejectionTest.java:664)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
</pre>
</details>

```
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.bitness: 64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64

```

--- a/engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceRejectionTest.java
+++ b/engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceRejectionTest.java
@@ -665,9 +665,9 @@ public class ModifyProcessInstanceRejectionTest {
             """
             Expected to modify instance of process '%s' \
             but it contains one or more activate instructions with an ancestor scope key \
-            that is not an ancestor of the element to activate:
-            - instance '%s' of element 'subProcess2' is not an ancestor of element 'A'
-            - instance '%s' of element 'B' is not an ancestor of element 'A'"""
+            that is not an ancestor of the element to activate:\
+            %n- instance '%s' of element 'subProcess2' is not an ancestor of element 'A'\
+            %n- instance '%s' of element 'B' is not an ancestor of element 'A'"""
                 .formatted(PROCESS_ID, subProcess2.getKey(), taskB.getKey()));
   }
 

